images/ruby/Alpine/supervisord.conf

```diff
@@ -0,0 +1,11 @@
+[supervisord]
+nodaemon=true
+logfile=/var/log/supervisor/supervisord.log ; (main log file;default $CWD/supervisord.log)
+
+[program:rails]
```

Comment:
Like php, we'd want to start 2 processes here: 
 - nginx - would serve static files and reverse proxy back to rails
 - rails (you ahve thi spart already)

Reply:
@jacobbeasley made the changes in the Alpine Dockerfile. please review.